eZ/Publish/Core/MVC/Symfony/Controller/Content/ViewController.php

```diff
@@ -99,6 +99,8 @@ protected function buildResponse($etag = null, DateTime $lastModified = null)
      * @throws \Exception
      *
      * @return \Symfony\Component\HttpFoundation\Response
+     *
+     * @deprecated Since 6.0.0. Viewing locations is now done with ViewContent.
```

Comment:
You also should use `trigger_error()` with `E_DEPRECATED_USER` (same for all deprecated methods in the controller)